test/stackTrace.js

```diff
@@ -37,10 +35,6 @@ tap.test('preserves stack trace with newlines', function (tt) {
             + 'not ok 1 Error: Preserve stack\n'
             + '  ---\n'
             + '    operator: error\n'
-            + '    expected: |-\n'
-            + '      undefined\n'
```

Comment:
Personally i do find this useful - it makes it clear what value is expected to make it pass.

Reply:
IMO, there isn't really an expected value, though... I think of `error` as like `fail`, where you call it to report that something went wrong, and the expected behavior is that it not get called at all. `expected: undefined` doesn't add any additional information on top of `operator: error`. If you find it useful anyway, hey, you do you, but I don't really understand why.